### PR TITLE
Update to latest

### PR DESCRIPTION
**Pull request by ErikSchierboom**
- exercises: update to latest distribution
- exercises: remove observability
- Update dependencies to latest

CC @glennj

### CODE REVIEW

File: exercises/practice/basics-monitor-rush/Ballerina.toml [hunk 1]
```diff
@@ -2,7 +2,4 @@
 org = "ballerina_exercism"
 name = "basics_monitor_rush"
 version = "0.1.0"
-distribution = "2201.2.3"
-
-[build-options]
-observabilityIncluded = true
```
Comment (ErikSchierboom) [inline exercises/practice/basics-monitor-rush/Ballerina.toml:8]: I'm also removing this, as I don't think we need it

File: exercises/practice/calculator-service/Dependencies.toml
Comment (ErikSchierboom) [inline exercises/practice/calculator-service/Dependencies.toml:1]: I'm unsure about what this file should be containing. Can we make it tinier? Do we need to include everything the student _could_ use?

**Comment by glennj** [comment]
For the failing exercises, I think they are status WIP. if you revert those changes the checks should pass

**Comment by ErikSchierboom** [comment]
@glennj I've opened https://github.com/exercism/ballerina/pull/179 which will remove the PR-specific workflow and thus `verify-exercises` will ignore `wip` exercises

**Comment by ErikSchierboom** [comment]
This has been rebased on https://github.com/exercism/ballerina/pull/179, which should be merged first.

**Review by kytrinyx — APPROVED** [review]

**Pull request merged**